### PR TITLE
GUI and Bug fixes pre-1.3.trial release

### PR DESCRIPTION
**Pull request by yashpola**
Font
- Increase font size for tags and status

Color
- Add orange-backing for `waiting list` statuses

Others
- Add labels for fields of `Applicant` in `PersonCard`
- Expand box sizes to prevent text cut-off

DG
- Fix typos
- Fix image sizes

Bug:
- `FilterPersonsByStatusCommand` check was missing in `AddressBookParser` leading to unknown command when attempting to `filter_by_status`

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -2,7 +2,7 @@
 
 ![Ui](docs/images/UpdatedUi.png)
 
-This is a professional Applicant Tracking System (ATS) for hiring managers in small to mid range startups.<br>
+Tether is a professional Applicant Tracking System (ATS) for hiring managers in small to mid range startups.<br>
```
Comment (gingerbreaf) [inline README.md:5]: Good job remembering to change the name.

File: docs/DeveloperGuide.md [hunk 2]
```diff
@@ -289,12 +288,13 @@ AddInterviewCommand implements the following operations:
...
 
-The above `excute` operation utilises `ModelManager#updateFilteredPersonList()` implemented from the Model interface to obtain the list of `applicant` and `interviewer` phone numbers.
-This is followed by checking the validity of the phone numbers before creating an `Interview` object to be added into the interview list. The operation `excute` then utilises
-`ModelManager#addInterview()` implemented from the Model to add the `Interview` to the list. The operation `excute` also utilises `ModelManager#sortInterview()` to the `interview` objects by `date`, `startTime` and `endTime`.
+The above `execute` operation utilises `ModelManager#updateFilteredPersonList()` implemented from the Model interface to obtain the list of `applicant` and `interviewer` phone numbers.
+This is followed by checking the validity of the phone numbers before creating an `Interview` object to be added into the interview list. The operation `execute` then utilises
+`ModelManager#addInterview()` implemented from the Model to add the `Interview` to the list. The operation `execute` also utilises `ModelManager#sortInterview()` to the `interview` objects by `date`, `startTime` and `endTime`.
```
Comment (gingerbreaf) [inline docs/DeveloperGuide.md:293]: Good job helping wesley correct his spelling.

File: src/main/resources/view/PersonListCard.fxml [hunk 3]
```diff
@@ -26,13 +26,13 @@
...
+        <Label fx:id="name" prefHeight="45.0" prefWidth="236.0" style="-fx-font-size: 30;" styleClass="cell_big_label" text="\$first" />
       </HBox>
       <FlowPane fx:id="tags" />
       <FlowPane fx:id="status" />
-      <Label fx:id="phone" prefHeight="19.0" prefWidth="110.0" style="-fx-font-size: 18;" styleClass="cell_small_label" text="\$phone" />
-      <Label fx:id="email" prefHeight="27.0" prefWidth="200.0" style="-fx-font-size: 18;" styleClass="cell_small_label" text="\$email" />
-      <Label fx:id="remark" prefHeight="27.0" prefWidth="200.0" style="-fx-font-size: 18;" styleClass="cell_small_label" text="\$remark" />
+      <Label fx:id="phone" prefHeight="23.0" prefWidth="234.0" style="-fx-font-size: 18;" styleClass="cell_small_label" text="\$phone" />
+      <Label fx:id="email" prefHeight="27.0" prefWidth="238.0" style="-fx-font-size: 18;" styleClass="cell_small_label" text="\$email" />
+      <Label fx:id="remark" prefHeight="27.0" prefWidth="234.0" style="-fx-font-size: 18;" styleClass="cell_small_label" text="\$remark" />
```
Comment (zhikaiong2001) [inline src/main/resources/view/PersonListCard.fxml:35]: Good job for increasing the width for visibility

**Review by zhikaiong2001 — APPROVED** [review]
LGTM

File: docs/index.md [hunk 4]
```diff
@@ -3,16 +3,21 @@
...
 
-**AddressBook is a desktop application for managing your contact details.** While it has a GUI, most of the user interactions happen using a CLI (Command Line Interface).
+Tether is a professional Applicant Tracking System (ATS) for hiring managers in small to mid range startups.<br>
 
-* If you are interested in using AddressBook, head over to the [_Quick Start_ section of the **User Guide**](UserGuide.html#quick-start).
+Hiring managers can use this application to keep track of the following:
+1. Applicant and Interviewer contacts
+2. Interview datetimes
+3. Application statuses
+
```
Comment (gingerbreaf) [inline docs/index.md:19]: Thanks for helping to edit this file!

**Review by gingerbreaf — APPROVED** [review]
Good work!

**Pull request merged**